Tests/Admin/Extension/LockExtensionTest.php

```diff
@@ -66,11 +66,18 @@ public function testConfigureFormFields()
     {
         $formMapper = $this->configureFormMapper();
         $form = $this->configureForm();
+
         $this->configureAdmin(null, null, $this->modelManager->reveal());
         $event = new FormEvent($form->reveal(), array());
 
         $this->modelManager->getLockVersion(array())->willReturn(1);
-        $form->add('_lock_version', 'hidden', array('mapped' => false, 'data' => 1))->shouldBeCalled();
+
+        // NEXT_MAJOR: remove the check and add the FQCN
+        $hiddenType = method_exists('Symfony\Component\Form\AbstractType', 'getBlockPrefix') ?
```

Comment:
I'm wondering whether this `method_exists` approach is performing well. I find this particular statement 115 times in SonataAdminBundle alone (including tests that is but still):

```
method_exists('Symfony\Component\Form\AbstractType', 'getBlockPrefix')
```

Shouldn't this be extracted as a global variable or something (out of scope of this PR obviously) or can we assume `method_exists` is optimized for this?